src/app/footer/footer.component.html

```diff
@@ -62,10 +62,11 @@ <h5 class="text-uppercase">Footer Content</h5>
         <a class="text-white"
            href="https://www.lyrasis.org/">{{ 'footer.link.lyrasis' | translate}}</a>
       </p>
-      <ul class="footer-info list-unstyled small d-flex justify-content-center mb-0">
+      <ul class="footer-info list-unstyled d-flex justify-content-center mb-0">
         <li>
-            <a class="text-white" href="javascript:void(0);"
-             (click)="showCookieSettings()">{{ 'footer.link.cookies' | translate}}</a>
+          <button class="btn btn-link text-white" type="button" (click)="showCookieSettings()">
```

Comment:
@alexandrevryghem : I'm still reviewing/testing this PR.  So far it looks great!  However, I have a question on commit https://github.com/DSpace/dspace-angular/pull/2683/commits/9ab30231aad9983cc2d961e639e4a12a22fe1633

That commit almost seems to go *against* normal accessibility (an example is on this line). Accessibility tends to say you should use the native elements as much as possible... and in this situation the native element is a link (`<a>`) but you've made a `<button>` that looks like a link.

If we want to go this route, these buttons all MUST have `role="link"` per https://accessibleweb.com/question-answer/is-it-ok-to-have-a-link-that-looks-like-a-button/

But overall, I'm not sure these should be made into buttons. See the accessibility descriptions between buttons and links: https://www.w3schools.com/accessibility/accessibility_buttons_links.php

Is there a reason why you chose to make "uncrawlable links" into buttons?  Maybe I'm missing how this somehow helps accessibility?

Reply:
Although... hmm.. right after I posted that question, I took a second look.  I think I see what you are doing.  You are only turning `<a>` into `<button>` when the link _is really an action that keeps you on the same page_.  So, that would be in line with the W3Schools definition above.  I may have convinced myself this is correct, but I'll leave this comment here in case others disagree.

Reply:
I'd say that both are confusing but the replacement is moving in the right direction.  The old code is a link which looks like a link but is hijacked to act like a button.  The new code is a button which acts like a button but looks like a link.  The old code is confusing to screen readers, while the new code is visually confusing.  If it acts as a button, it should be coded `<button/>` and look like a button.

Reply:
@mwoodiupui : While I understand your point, I think the UI will look awkward if we make these all look like buttons.  In this specific scenario (where these inline comments are), it's modifying the links in our default footer (see footer at https://demo.dspace.org).

By default there are 4 links there:
* Cookie settings (changed to a `<button>` in this PR, as you don't leave the page you are on to edit cookie settings)
* Privacy Policy (remains an `<a>` link, as it sends you to a different page)
* End User Agreement (remains an `<a>` link, as it sends you to a different page)
* Send Feedback (remains an `<a>` link, as it sends you to a different page)

Changing "Cookie Settings" to visually look like a button is awkward in this footer location.  It would be possible, but would likely require further redesign of the footer (which is not the intent of this PR).

Overall, in summary, I now see why @alexandrevryghem made the changes in https://github.com/DSpace/dspace-angular/commit/9ab30231aad9983cc2d961e639e4a12a22fe1633.  I'm OK with them. I do agree there may be more we can do here, but it likely is out-of-scope for this PR.